inginious/frontend/user_manager.py

```diff
@@ -33,6 +33,14 @@ class AuthInvalidMethodException(Exception):
 
 class AuthMethod(object, metaclass=ABCMeta):
 
+    @abstractmethod
```

Comment:
Those methods do not have to be defined in `AuthMethod` class as there should be no dependance between them.